subworkflows/nf-neuro/anatomical_segmentation/main.nf

```diff
@@ -15,6 +19,10 @@ workflow ANATOMICAL_SEGMENTATION {
 
         ch_versions = Channel.empty()
 
+        if ( ch_image && ch_freesurferseg ) {
+            error('Both input channels cannot be passed simultaneously')
+        }
```

Comment:
Remove this. There is no problem in having both supplied. Behavior is controlled through `params` via `if/else` statements. There is no point in restricting this here.